lte/protos/session_manager.proto

```diff
@@ -52,12 +54,16 @@ message CommonSessionContext {
   string apn = 4;
   bytes msisdn = 5;
   RATType rat_type = 6;
+  //PDU session state to mirror with AMF or MME
+  SMSessionFSMState sm_session_state = 7;
+  uint32 sm_session_version = 9;
```

Comment:
shouldnt this be part of the `M5GSMSessionContext`

Reply:
After discussing with Marie, it had been concluded, It should be part of common context.

Reply:
@uri200 the session version + session state will be used later on when we migrate 4G to a set interface.

Reply:
@uri200 , I can resolve this comment.